src/sentiment.py

```diff
@@ -22,10 +22,15 @@ def __init__(self, file):
         return
 
     def analyze(self, text):
-        return self.analyzer.analyze(
+        if text == "":
```

Comment:
the string will literally be "empty", so this should be text == "empty". I was worried having an actual empty line would mess up the reading so i put "empty" instead

Reply:
lmao that is huge brained